samples/nrf9160/gnss/src/main.c

```diff
@@ -241,6 +278,11 @@ static void ttff_test_got_fix_work_fn(struct k_work *item)
 	if (time_blocked > 0) {
 		LOG_INF("Time GNSS was blocked by LTE: %u", time_blocked);
 	}
+	if (ref_used) {
+		LOG_INF("Distance from reference: %.01f",
```

Comment:
Or you could call print_distance_from_reference() here?

Reply:
The reason why I didn't call that was that the function used `printk()`, while here I wanted to use logger. Now that I wrote this comment, I realized that I could have just selected between `printk()` and `LOG_INF()` in `print_distance_from_reference()` based on the configuration.